### PR TITLE
DOCS-162 - Add new UI hidden instructions to Metrics (Kanso)

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request adds Kanso instructions to articles in this section:
https://help.sumologic.com/docs/metrics/

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/metrics/logs-to-metrics.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ This section describes how to create a Logs-to-Metrics rule.
 You must be a Sumo admin to create a Logs-to-Metrics rule, or have a role with the **Manage Logs-to-Metrics** capability, as described in the previous section.
 :::
 
-1. Go to **Manage Data > Metrics > Logs-to-Metrics** in the Sumo web app. The page displays a list of existing Logs-to-Metrics rules. <br/><img src={useBaseUrl('img/metrics/log-to-metrics-add.png')} alt="Add button" style={{border: '1px solid gray'}} width="800" />
+1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Metrics > Logs-to-Metrics**. <!--Kanso [**New UI**](/docs/get-started/sumo-logic-ui-new/). In the top menu select **Configuration**, and then under **Metrics** select **Logs to Metrics**. You can also click the **Go To...** menu at the top of the screen and select **Logs to Metrics**. Kanso--> <br/>The page displays a list of existing Logs-to-Metrics rules. <br/><img src={useBaseUrl('img/metrics/log-to-metrics-add.png')} alt="Add button" style={{border: '1px solid gray'}} width="800" />
```
Comment (kimsauce) [inline docs/metrics/logs-to-metrics.md:109]:
```suggestion
1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Metrics > Logs-to-Metrics**. <!--Kanso [**New UI**](/docs/get-started/sumo-logic-ui-new/). In the top menu, select **Configuration**, and then under **Metrics**, select **Logs to Metrics**. You can also click the **Go To...** menu at the top of the screen and select **Logs to Metrics**. Kanso--> <br/>The page displays a list of existing Logs-to-Metrics rules. <br/><img src={useBaseUrl('img/metrics/log-to-metrics-add.png')} alt="Add button" style={{border: '1px solid gray'}} width="800" />
```

File: docs/metrics/introduction/index.md [hunk 2]
```diff
@@ -86,7 +86,13 @@ For more information, see [About Metrics Rules](/docs/metrics/metric-rules-edito
...
 ## Metrics queries
 
-To query metrics in Sumo, you open the Metrics Explorer by clicking **+ New**, and selecting **Metrics**. In the screenshot below, the Metrics Explorer show the results of two queries.
+<!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso-->  To access Metrics Explorer, click the **+ New** button at the top of the screen and select **Metrics**. 
+
+<!--Kanso 
+[**New UI**](/docs/get-started/sumo-logic-ui-new/). To access Metrics Explorer, in the main Sumo Logic menu select **Metrics > Metrics Search**. You can also click the **Go To...** menu at the top of the screen and select **Metrics Search**. 
+ Kanso-->
+
+In the screenshot below, the Metrics Explorer show the results of two queries.
```
Comment (kimsauce) [inline docs/metrics/introduction/index.md:95]:
```suggestion
In the screenshot below, the Metrics Explorer shows the results of two queries.
```

File: docs/metrics/manage-metric-volume/disabled-metrics-sources.md [hunk 3]
```diff
@@ -93,7 +93,13 @@ This limit does not apply to the Kubernetes dimensions, Docker dimensions, or th
 * `label.com.docker.swarm.task.id`
 * `label.com.docker.swarm.task.name`
 
-For other dimensions that exceed the cardinality limit Sumo generates a metric transformation rule to exclude the dimension. The rule appears in the UI at **Manage Data > Metrics > Metric Transformation Rules**:
+For other dimensions that exceed the cardinality limit Sumo generates a metric transformation rule to exclude the dimension. The rule appears in the UI in **Metrics Transformation Rules**.
```
Comment (kimsauce) [inline docs/metrics/manage-metric-volume/disabled-metrics-sources.md:96]:
```suggestion
For other dimensions that exceed the cardinality limit, Sumo generates a metric transformation rule to exclude the dimension. The rule appears in the UI in **Metrics Transformation Rules**.
```

File: docs/metrics/manage-metric-volume/disabled-metrics-sources.md [hunk 4]
```diff
@@ -93,7 +93,13 @@ This limit does not apply to the Kubernetes dimensions, Docker dimensions, or th
...
+
+<!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso-->  To access **Metrics Transformation Rules**, in the main Sumo Logic menu select **Manage Data > Metrics > Metrics Transformation Rules**.
```
Comment (kimsauce) [inline docs/metrics/manage-metric-volume/disabled-metrics-sources.md:98]:
```suggestion
<!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso-->  To access **Metrics Transformation Rules**, go to the main Sumo Logic menu and select **Manage Data > Metrics > Metrics Transformation Rules**. 
```

File: docs/metrics/metric-rules-editor.md [hunk 5]
```diff
@@ -82,7 +88,8 @@ Different classes have dot-delimited names of various lengths, so a match expres
 
 Here's how to create a metrics rule using the metrics rules editor in the Sumo UI:
 
-1. In the Sumo web app, go to **Manage Data > Metrics > Metrics Rules**. The **Metrics Rules** page lists the metrics rules that have already been defined.<br/><img src={useBaseUrl('img/metrics/named-rule.png')} alt="Metrics rules" style={{border: '1px solid gray'}} width="800" />
+1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Metrics > Metrics Rules**. <!--Kanso 
+[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the top menu select **Configuration**, and then under **Metrics** select **Metrics Rules**. You can also click the **Go To...** menu at the top of the screen and select **Metrics Rules**. Kanso--><br/>The **Metrics Rules** page lists the metrics rules that have already been defined.<br/><img src={useBaseUrl('img/metrics/named-rule.png')} alt="Metrics rules" style={{border: '1px solid gray'}} width="800" />
```
Comment (kimsauce) [inline docs/metrics/metric-rules-editor.md:92]:
```suggestion
[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the top menu, select **Configuration**, and then under **Metrics**, select **Metrics Rules**. Alternatively, you can click the **Go To...** menu at the top of the screen and select **Metrics Rules**. Kanso--><br/>The **Metrics Rules** page lists the metrics rules that have already been defined.<br/><img src={useBaseUrl('img/metrics/named-rule.png')} alt="Metrics rules" style={{border: '1px solid gray'}} width="800" />
```

File: docs/metrics/metrics-queries/metrics-explorer.md [hunk 6]
```diff
@@ -46,7 +46,7 @@ If the query editor mode cannot be adjusted, it is changed to advanced mode so t
 
 ## Create a metric query
 
-1. In the Sumo Logic UI, click **+ New** and choose **Metrics** from the dropdown list.
+1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso-->  Click the **+ New** button at the top of the screen and select **Metrics**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the main Sumo Logic menu select **Metrics > Metrics Search**. You can also click the **Go To...** menu at the top of the screen and select **Metrics Search**.  Kanso-->
```
Comment (kimsauce) [inline docs/metrics/metrics-queries/metrics-explorer.md:49]:
```suggestion
1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso-->  Click the **+ New** button at the top of the screen and select **Metrics**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the main Sumo Logic menu, select **Metrics > Metrics Search**. Alternatively, you can click the **Go To...** menu at the top of the screen and select **Metrics Search**.  Kanso-->
```

File: docs/metrics/metrics-transformation-rules.md [hunk 7]
```diff
@@ -34,7 +34,7 @@ Metrics transformation rules are useful when:
 
 ## Create a Metrics Transformation Rule
 
-1. Navigate to **Manage Data > Metrics > Metrics Transformation Rules**. 
+1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Metrics > Metrics Transformation Rules**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the top menu select **Configuration**, and then under **Metrics** select **Metrics Transformation Rules**. You can also click the **Go To...** menu at the top of the screen and select **Metrics Transformation Rules**. Kanso-->
```
Comment (kimsauce) [inline docs/metrics/metrics-transformation-rules.md:37]:
```suggestion
1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Metrics > Metrics Transformation Rules**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the top menu, select **Configuration**, and then under **Metrics** select **Metrics Transformation Rules**. Alternatively, you can click the **Go To...** menu at the top of the screen and select **Metrics Transformation Rules**. Kanso-->
```

File: docs/metrics/metrics-transformation-rules.md [hunk 8]
```diff
@@ -62,13 +62,13 @@ Metrics transformation rules are useful when:
 
 ## Edit a metrics transformation rule
 
-1. Navigate to **Manage Data > Metrics > Metrics Transformation Rules**. 
+1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Metrics > Metrics Transformation Rules**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the top menu select **Configuration**, and then under **Metrics** select **Metrics Transformation Rules**. You can also click the **Go To...** menu at the top of the screen and select **Metrics Transformation Rules**. Kanso-->
```
Comment (kimsauce) [inline docs/metrics/metrics-transformation-rules.md:65]:
```suggestion
1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Metrics > Metrics Transformation Rules**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the top menu, select **Configuration**, and then under **Metrics** select **Metrics Transformation Rules**. Alternatively, you can click the **Go To...** menu at the top of the screen and select **Metrics Transformation Rules**. Kanso-->
```

File: docs/metrics/metrics-transformation-rules.md [hunk 9]
```diff
@@ -62,13 +62,13 @@ Metrics transformation rules are useful when:
...
 1. Click the rule you want to edit.
 1. Click **Edit** in the right hand pane. 
 
 ## Delete a metrics transformation rule
 
-1. Navigate to **Manage Data > Metrics > Metrics Transformation Rules**. 
+1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Metrics > Metrics Transformation Rules**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the top menu select **Configuration**, and then under **Metrics** select **Metrics Transformation Rules**. You can also click the **Go To...** menu at the top of the screen and select **Metrics Transformation Rules**. Kanso-->
```
Comment (kimsauce) [inline docs/metrics/metrics-transformation-rules.md:71]:
```suggestion
1. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Metrics > Metrics Transformation Rules**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the top menu, select **Configuration**, and then under **Metrics**, select **Metrics Transformation Rules**. Alternatively, you can click the **Go To...** menu at the top of the screen and select **Metrics Transformation Rules**. Kanso--> 
```

File: docs/metrics/manage-metric-volume/disabled-metrics-sources.md [hunk 10]
```diff
@@ -93,7 +93,13 @@ This limit does not apply to the Kubernetes dimensions, Docker dimensions, or th
 * `label.com.docker.swarm.task.id`
 * `label.com.docker.swarm.task.name`
 
-For other dimensions that exceed the cardinality limit Sumo generates a metric transformation rule to exclude the dimension. The rule appears in the UI at **Manage Data > Metrics > Metric Transformation Rules**:
+For other dimensions that exceed the cardinality limit Sumo generates a metric transformation rule to exclude the dimension. The rule appears in the UI in **Metrics Transformation Rules**.
+
+<!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso-->  To access **Metrics Transformation Rules**, in the main Sumo Logic menu select **Manage Data > Metrics > Metrics Transformation Rules**. 
+
+<!--Kanso 
+[**New UI**](/docs/get-started/sumo-logic-ui-new/). To access **Metrics Transformation Rules**, in the top menu select **Configuration** and then under **Metrics** select **Metrics Transformation Rules**. You can also click the **Go To...** menu at the top of the screen and select **Metrics Transformation Rules**.
```
Comment (kimsauce) [inline docs/metrics/manage-metric-volume/disabled-metrics-sources.md:101]:
```suggestion
[**New UI**](/docs/get-started/sumo-logic-ui-new/). To access **Metrics Transformation Rules**, go to the top menu select **Configuration**, and then under **Metrics**, select **Metrics Transformation Rules**. Alternatively, you can click the **Go To...** menu at the top of the screen and select **Metrics Transformation Rules**. 
```

File: docs/metrics/manage-metric-volume/data-limits-for-metrics.md [hunk 11]
```diff
@@ -13,7 +13,13 @@ For information on metrics retention, see [Metric Ingestion and Storage](metric-
...
 
-For billing and reporting purposes, data volume for metrics is measured in Data Points per Minute (DPM). When the DPM limit is exceeded, data is cached on the host and the Source is throttled. To check your current account usage relative to your available DPM limit, see the **Account Overview** tab of the Account page, at **Administration > Account**.
+For billing and reporting purposes, data volume for metrics is measured in Data Points per Minute (DPM). When the DPM limit is exceeded, data is cached on the host and the Source is throttled. To check your current account usage relative to your available DPM limit, see the **Account Overview** tab.  
+
+<!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso-->  To access the **Account Overview** tab, in the main Sumo Logic menu select **Administration > Account**. 
+
+<!--Kanso 
+[**New UI**](/docs/get-started/sumo-logic-ui-new/). To access the **Account Overview** tab, in the top menu select **Administration** and then under **Account** select **Account Overview**. You can also click the **Go To...** menu at the top of the screen and select **Account Overview**.
```
Comment (kimsauce) [inline docs/metrics/manage-metric-volume/data-limits-for-metrics.md:21]:
```suggestion
[**New UI**](/docs/get-started/sumo-logic-ui-new/). To access the **Account Overview** tab, go to the top menu select **Administration**, and then under **Account**, select **Account Overview**. Alternatively, you can click the **Go To...** menu at the top of the screen and select **Account Overview**. 
```

**Review by kimsauce — APPROVED** [review]
minor stuff!

File: docs/metrics/metric-rules-editor.md (hunk 5 above)
Comment (jpipkin1) [reply docs/metrics/metric-rules-editor.md:92]: I'll leave this as-is since it's consistent with many other entries.

**Comment by jpipkin1** [comment]
Thanks for the review, @kimsauce! I decided to leave as-is all the instances that say "You can also click..." since it's three fewer syllables than "Alternatively, you can click...".

**Pull request merged**